src/formatters/PropertyListFormatter.cc

```diff
@@ -0,0 +1,157 @@
+// -*- lsst-c++ -*-
+
+/*
+ * LSST Data Management System
+ * Copyright 2008, 2009, 2010 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+
+/** @file
+ * @brief Implementation of PropertyListFormatter class
+ *
+ * @ingroup afw
+ */
+
+#ifndef __GNUC__
+#  define __attribute__(x) /*NOTHING*/
+#endif
+static char const* SVNid __attribute__((unused)) =
+    "$Id$";
```

Comment:
Yes; I didn't remove it as it was an unnecessary change to the` DecoratedImageFormatter`, but that hardly matters in a copy.